doc/api/fs.md

```diff
@@ -685,6 +685,33 @@ added: v10.0.0
 
 Changes the ownership of a file.
 
+### `fsPromises.copy(src, dest[, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `src` {string|Buffer|URL} source path to copy.
+* `dest` {string|Buffer|URL} destination path to copy to.
+* `options` {Object|Function}
+  * `dereference` {boolean} dereference symlinks. **Default:** `false`.
+  * `errorOnExist` {boolean} when `force` is `false`, and the destination
+    exists, throw an error. **Default:** `false`.
+  * `filter` {Function} Function to filter copied files/directories. Return
```

Comment:
Can this be an async function? Experience shows that if it can take a function, users *will* pass an async function...

Reply:
It can be:

```js
async function handleFilter(onInclude, destStat, src, dest, opts, cb) {
  const include = await opts.filter(src, dest);
  if (include) return onInclude(destStat, src, dest, opts, cb);
}
```

Reply:
It would be good to call that out in the docs.